details.py

```diff
@@ -1,110 +1,162 @@
-#!/usr/bin/env python3
```

Comment:
I love it!
My only "issue" is the shebang: I usually use virtual environments, making `#!/usr/bin/env python3` almost a necessity, but I can run it with `python3 ...`.